cli/buildengine.ts

```diff
@@ -387,14 +387,29 @@ function runDockerAsync(args: string[]) {
         if (process.platform == "darwin")
             mountArg += ":delegated"
 
+        let fullArgs = ["--rm", "-v", mountArg, "-w", "/src"].concat(dargs).concat([cs.dockerImage]).concat(args);
```

Comment:
All of these `concat`s look a little funky to me. Can we use spread syntax here instead?

Reply:
sure